must-gather/Dockerfile

```diff
@@ -1,3 +1,14 @@
+FROM quay.io/konveyor/builder:latest as konveyor-builder
+ARG RESTIC_BRANCH=konveyor-0.15.0
```

Comment:
Why is this not konveyor-dev also?

for cherrypicking to oadp-1.1 and oadp-1.2 we can change this to oadp-1.2 and oadp-1.1 respectively.

Reply:
The build commands came from openshift/velero, where it is set to 0.15.0 temporarily: https://github.com/openshift/velero/commit/08bf989803b8c69a516da1a19ac3a6f83c5bb9f7

Reply:
ok cool.